cpp/ql/test/experimental/library-tests/rangeanalysis/extensibility/extensibility.ql

```diff
@@ -1,5 +1,7 @@
 import semmle.code.cpp.rangeanalysis.SimpleRangeAnalysis
+import semmle.code.cpp.rangeanalysis.RangeAnalysisUtils // for typeLowerBound
```

Comment:
We don't usually place comments on imports stating why we're importing them. Is there a particular reason we're doing this here?

Reply:
Not really. Removed.